wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -554,6 +554,46 @@ def icons():
     return {'icons': icons}
 
 
+@register.simple_tag
+def format_collection(coll: Collection, min_depth: int = 2) -> str:
```

Comment:
Will that work, considering that django templates can't send arguments to function calls?

Reply:
Sorry, yes. I could have been a bit clearer there. The tag is fine to stay... It will just become a means to call the new model method with a different `indentation_start_depth` value.

Reply:
Ahhh, OK. That makes sense. Will do.